packages/devtools_app/lib/src/shared/config_specific/host_platform/host_platform_stub.dart

```diff
@@ -3,7 +3,7 @@
 // in the LICENSE file.
 
 class HostPlatform {
-  static late final HostPlatform instance = HostPlatform();
```

Comment:
same question with the Host platform files - why remove late?

Reply:
same answer: https://github.com/dart-lang/linter/issues/4307